context/app/static/js/components/Detail/Summary/Summary.stories.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import Typography from '@material-ui/core/Typography';
+
+import { Default as DefaultCitation } from 'js/components/Detail/Citation/Citation.stories';
+import Summary from './Summary';
+import SummaryItem from '../SummaryItem';
+
+export default {
+  title: 'EntityDetail/Summary/Summary',
+  component: Summary,
+};
+
+const sharedArgs = {
+  uuid: 'fakeuuid',
+  create_timestamp: Date.now(),
+  last_modified_timestamp: Date.now(),
+  display_doi: 'DOI123',
+};
+
+const donorSharedArgs = {
+  ...sharedArgs,
+  entity_type: 'Donor',
+};
+
+const ChildlessTemplate = (args) => <Summary {...args} />;
+export const DonorDefault = ChildlessTemplate.bind({});
+DonorDefault.args = {
+  ...donorSharedArgs,
+};
+
+export const DonorWithDescription = ChildlessTemplate.bind({});
+DonorWithDescription.args = {
+  ...donorSharedArgs,
+  description:
+    'Fugiat irure nisi ea dolore non adipisicing non. Enim enim incididunt ut reprehenderit esse sint adipisicing. Aliqua excepteur reprehenderit tempor commodo anim veniam laboris labore exercitation qui. Adipisicing pariatur est anim nisi cupidatat ea Lorem nostrud labore laborum enim eiusmod.',
+};
```

Comment:
```suggestion
const lorem = 'Fugiat irure nisi ea dolore non adipisicing non. Enim enim incididunt ut reprehenderit esse sint adipisicing. Aliqua excepteur reprehenderit tempor commodo anim veniam laboris labore exercitation qui. Adipisicing pariatur est anim nisi cupidatat ea Lorem nostrud labore laborum enim eiusmod.'

export const DonorWithDescription = ChildlessTemplate.bind({});
DonorWithDescription.args = {
  ...donorSharedArgs,
  description: lorem,
};
```
and similarly below.